src/solvers/td3.ml

```diff
@@ -593,7 +602,10 @@ module WP =
                 if tracing then trace "sol2" "stable remove %a\n" S.Var.pretty_trace y;
                 HM.remove stable y;
                 HM.remove superstable y;
-                destabilize_with_side ~front:false y
+                if restart_destab_with_sides then
+                  destabilize_with_side ~front:false y
+                else
+                  destabilize_normal ~front:false y
```

Comment:
This seems a lot like restarting with 1 fuel (#629) in that it only continues destabilizing without restarting sides. Although here it's a bit differently checked at `side_dep` rather than `side_infl`.

For the purposes of some last-minute benchmarking, this is fine and we can afterwards decide whether to maybe integrate this with #629 or strip out entirely.